distributed/worker.py

```diff
@@ -1653,26 +1653,26 @@ async def get_data(
 
         if self.status == Status.paused:
             max_connections = 1
-            throttle_msg = " Throttling outgoing connections because worker is paused."
+            throttle_msg = " Throttling incoming connections because worker is paused."
```

Comment:
Fixing `throttle_msg`.

Reply:
These are inbound connections for outbound data. Maybe change it to "Throttling outgoing data transfers"?

Reply:
Done